cooldowns_shaman.lua

```diff
@@ -258,7 +258,15 @@ LCT_SpellData[204366] = {
 	talent = true,
 	cooldown = 45,
 }
--- Bloodlust
+-- Shamanism (Alliance)

```

Comment:
Not sure why bloodlust / heroism is not working. I tried to use the spell ID for pvp talent shamanism as well, didn't work.

Reply:
Everything else is tested and working properly.

Reply:
Will try to take a look later

Reply:
Perhaps need to put a baseline bloodlust + heroism, and put the talent one with condition "replace ="

Reply:
can also have simple aliases

Reply:
Finally found why it doesn't work, the PVP talent modifies the ability to a new one with different spell ids, now it works perfectly.
![image](https://user-images.githubusercontent.com/78008331/107895533-1049de80-6f77-11eb-8d2a-3c2150886a32.png)